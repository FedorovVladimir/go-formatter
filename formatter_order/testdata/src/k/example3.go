package main

func f2() { // want "formatter_order"
}

// c - c // want "formatter_order"
const c = "c"

// c2 - c2 // want "formatter_order"
const (
	c2 = "c2"
)
