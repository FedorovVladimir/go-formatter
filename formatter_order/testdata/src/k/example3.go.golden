package main

// c - c // want "incorrect declaration order"
const c = "c"

// c2 - c2 // want "incorrect declaration order"
const (
	c2 = "c2"
)

func f2() { // want "incorrect declaration order"
}
